### ISSUE TITLE
Reduced respiratory ciliary beating frequency

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6202]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2658692/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/27486783/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
